test-harness/specs/deprecated-keyword.only_oas3.txt

```diff
@@ -0,0 +1,31 @@
+====test====
+Prism does not report violations of deprecated parameters specification as errors.
+====spec====
+openapi: 3.0.2
+paths:
+  /test:
+    get:
+      parameters:
+        - deprecated: true
+          in: query
+          name: productId
+      responses:
+        200:
+          description: successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  prop:
+                    type: string
+                    example: abc
+====server====
+mock -p 4010 ${document}
+====command====
+curl -i http://localhost:4010/test
```

Comment:
`productId` is not used and it still reports a deprecation warning. I believe it should not report deprecation in such case.

Reply:
Testing the deprecation warning is hard unfortunately (because it's a side effect in Prism — and it should not).

When one day we'll introduce the `IO` abstraction from fp-ts this won't be a problem anymore. For now we will have to stick with what we have.